Programs/_freeze_module.py

```diff
@@ -0,0 +1,69 @@
+"""Python implementation of Programs/_freeze_module.c
+
+The pure Python implementation uses same functions and arguments as the C
+implementation.
+
+The generated byte code is slightly different because
+compile() sets the PyCF_SOURCE_IS_UTF8 flag and objects have a
+reference count > 1. Marshal adds the `FLAG_REF` flag and creates a
+reference `hashtable`.
```

Comment:
Yeah, this caused me a big headache at one point because on debug builds some of the frozen strings had multiple refs where they only had 1 on non-debug builds, so the resulting marshaled data was different.  We solved that but it was a pain.  Thanks for reminding me. 😄